docs/services/vision/segmentation.md

```diff
@@ -121,10 +121,7 @@ The following parameters are available for a `"obstacles_pointcloud"`.
 | --------- | --------- | ----------- |
 | `min_points_in_plane` | Optional | An integer that specifies how many points there must be in a flat surface for it to count as a plane. This is to distinguish between large planes, like the floors and walls, and small planes, like the tops of bottle caps. <br> Default: `500` </br> |
```

Comment:
I meant it here in "min_points_in_plane" don't hate me lol

Reply:
@kharijarrett Hated (/s) 🤣 I updated but lmk what you think of the update to `min_points_in_plane`, I'm still not sure if I'm phrasing it well!